lib/linux_admin/security/modprobe.rb

```diff
@@ -0,0 +1,53 @@
+module LinuxAdmin
+  class Security
+    class Modprobe
+      CONF_FILE_NAME = "/etc/modprobe.d/scap_lockdown"
```

Comment:
Let's make the constant name consistent across the classes...For example in LoginDefs it's called  CONF_FILE